src/TxHistory/BalanceBanner.tsx

```diff
@@ -75,7 +77,7 @@ const PairedBalance = ({privacyMode}: {privacyMode: boolean}) => {
   const pairedTotal = privacyMode ? hiddenPairedTotal : total
 
   return (
-    <Text style={styles.totalText}>
+    <Text style={styles.totalText} testID="fiatConvertedBalanceText">
```

Comment:
Just to keep consistency
```suggestion
    <Text style={styles.totalText} testID="pairedTotalText">
```

Reply:
Changed